notebooks/setfit-optimum-intel.ipynb

```diff
@@ -0,0 +1,977 @@
+{
+    "cells": [
+        {
+            "cell_type": "markdown",
+            "id": "76571396-8f54-40ed-9e81-6c7531e6eaee",
+            "metadata": {
+                "id": "76571396-8f54-40ed-9e81-6c7531e6eaee"
+            },
+            "source": [
+                "# Accelerate SetFit Models with [`optimum-intel`](https://github.com/huggingface/optimum-intel)"
+            ]
+        },
+        {
+            "cell_type": "markdown",
+            "id": "4733f283-4736-4d9d-854e-5582280b074d",
+            "metadata": {},
+            "source": [
+                "In this notebook, we'll show how you can use the 🤗 `optimum-intel` package to speed up your SetFit models on **Intel CPUs**.\\\n",
+                "In our experiments, we will be using a fine-tuned SetFit model based on `bge-small-en-v1.5` and observe our performance increase."
+            ]
+        },
+        {
+            "cell_type": "markdown",
+            "id": "d4099797-55d8-492a-94d4-4d815e72aeb8",
+            "metadata": {},
+            "source": [
+                "#### Note\n",
+                "To reproduce the maximal speeds reported in this notebook, launch it locally, using `numactl`.\\\n",
```

Comment:
```suggestion
                "To reproduce the maximum speed gains reported in this notebook, please launch it locally, using `numactl`.\\\n",
```

Reply:
thanks @orenpereg 
What I mean here is the absolute latencies, not necessarily the speedups.